gdal_subsetter/transform.py

```diff
@@ -1,42 +1,40 @@
-"""
- CLI for adapting a Harmony operation to GDAL
+""" CLI for adapting a Harmony operation to GDAL
```

Comment:
This difference looks busy. Mostly that's due to whitespace clean-up (thanks vim!) and tidying up import statements so that we aren't importing large bits of packages we aren't using.